doc/user/isisd.rst

```diff
@@ -203,6 +203,10 @@ ISIS interface
 
    Add padding to IS-IS hello packets.
 
+.. clicmd:: isis hello padding sometimes
```

Comment:
I feel the keyword sometimes here is not very clear. It's better to use something meaningful may be something like "during-adjacency-formation"

Reply:
yeah I'm not a fan of `isis hello padding sometimes`  We need something clearer

Reply:
I agree. I was so used to cisco's and juniper's syntax that I forgot that it's not that clear. Changed it to your suggestion @mobash-rasool, thanks!